Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -237,6 +238,9 @@ static bool ShouldDisplayGameListItem(const GameListItem& item)
   if (!show_platform)
     return false;
 
+  if (item.GetPlatform() == DiscIO::Platform::WII_WAD && !IOS::ES::IsChannel(item.GetTitleID()))
+    return false;
```

Comment:
Oh, I see what you mean. Returning early from the constructor would cause the cache entry to not be created, so we would need to scan these WADs again on the next scan.